### PR TITLE
feat: show ENS username details

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: ui/app/AppLayouts/Profile/Sections/Ens/ENSDetails.qml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            if(username != ensName) return;
+                walletAddressLbl.text = address;
+                walletAddressLbl.textToCopy = address;
+                keyLbl.text = pubkey.substring(0, 20) + "..." + pubkey.substring(pubkey.length - 20);
+                keyLbl.textToCopy = pubkey;
+                walletAddressLbl.visible = true;
+                keyLbl.visible = true;
+        }
+        onLoading: {
+            console.log("ABC")
```
Comment (emizzle) [inline ui/app/AppLayouts/Profile/Sections/Ens/ENSDetails.qml:51]: 🔪

File: ui/app/AppLayouts/Profile/Sections/Ens/List.qml [hunk 2]
```diff
@@ -7,6 +7,7 @@ import "../../../Chat/ChatColumn/MessageComponents"
 
 Item {
     property var onClick: function(){}
+    property var onSelectENS: function(){}
```
Comment (emizzle) [inline ui/app/AppLayouts/Profile/Sections/Ens/List.qml:10]: Is this better than using a signal?
```
signal selectEns(string username)
```
Which can be used like:
```
List {
    onSelectEns: {
        console.log(“username:”, username)
    }
}
```

Comment (richard-ramos) [reply ui/app/AppLayouts/Profile/Sections/Ens/List.qml:10]: I did not know that was possible

File: ui/app/AppLayouts/Profile/Sections/Ens/ENSDetails.qml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    }
+
+    TextWithLabel {
+        id: walletAddressLbl
+        label: qsTr("Wallet address")
+        visible: false
+        text:  ""
+        textToCopy: ""
+        anchors.left: parent.left
+        anchors.leftMargin: 24
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/ENSDetails.qml:66]: 24 is `Style.current.bigPadding`

File: ui/app/AppLayouts/Profile/Sections/Ens/ENSPopup.qml [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+            }
+        }
+    }
+
+    onNewUsernameChanged: {
+        btnSelectPreferred.state = newUsername === profileModel.ens.preferredUsername ? "inactive" : "active"
+    }
+    
+    footer: Item {
+        anchors.top: parent.bottom
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/ENSPopup.qml:72]: Was that one voluntary? Did you mean `parent.top`?

File: ui/shared/TextWithLabel.qml [hunk 5]
```diff
@@ -25,7 +26,6 @@ Item {
         id: textItem
         text: infoText.text
         font.family: fontFamily
-        selectByMouse: true
```
Comment (jrainville) [inline ui/shared/TextWithLabel.qml:28]: Why remove this?

**Pull request merged**